handout/forms/form-builder/form-builder_basics.md

```diff
@@ -16,14 +16,14 @@ import {
 
 @Component({
   selector: 'login-form',
-  templateUrl: 'app/login-form.component.html',
```

Comment:
@smithad15 Rangle (at least at one point) was in favour of trailing commas because they simplify diffs: https://github.com/rangle/react-starter/blob/master/.eslintrc#L39 I don't know why this did not make it into our angular 2 starter: https://github.com/rangle/angular2-starter/blob/master/tslint.json#L52 There's other supporting Rangle documentation about the dangling comma issue.  I'm generally opposed to trailing commas but I find the git diff argument persuasive.